diagnosis/README.md

```diff
@@ -4,9 +4,13 @@ This short [guide](https://github.com/Azure/aks-engine/blob/master/docs/howto/tr
 
 Please refer to this [article](https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-solution-template-kubernetes-trouble) for specifics about how the `Kubernetes Cluster` marketplace item works on Azure Stack.
 
+## Prerequisites:
+1.	Enable Azure CLI on AzureStack.  Please refer to this [article](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-version-profiles-azurecli2?view=azs-1908) 
+2.	Use az login –service-principal -u <spn-client-id> -p <spn-client-password> --tenant <tenant-id>
+
 ## Gathering logs
 
-The Bash scripts on this directory are aim to simplify the collection of relevant logs from your Kubernetes cluster. Instead of SSH-ing into the cluster nodes, you can simply download and execute script `getkuberneteslogs.sh` and wait for the logs to be saved back into your workstation.  
+The Bash scripts on this directory are aim to simplify the collection of relevant logs from your Kubernetes cluster. Instead of SSH-ing into the cluster nodes, you can simply download zip file and execute script `getkuberneteslogs.sh` and wait for the logs to be saved back into your workstation.  
```

Comment:
This may be better: `download and extract the latest [release](link-to-release) and execute script`